src/policy/fees.h

```diff
@@ -163,25 +163,26 @@ class CBlockPolicyEstimator
     static constexpr double SUFFICIENT_TXS_SHORT = 0.5;
 
     /** Minimum and Maximum values for tracking feerates
-     * The MIN_BUCKET_FEERATE should just be set to the lowest reasonable feerate we
-     * might ever want to track.  Historically this has been 1000 since it was
-     * inheriting DEFAULT_MIN_RELAY_TX_FEE and changing it is disruptive as it
-     * invalidates old estimates files. So leave it at 1000 unless it becomes
-     * necessary to lower it, and then lower it substantially.
+     * The MIN_BUCKET_FEERATE and MAX_BUCKET_FEERATE should just be set to
+     * the lowest and highest reasonable feerate we might ever want to track.
      */
-    static constexpr double MIN_BUCKET_FEERATE = 1000;
```

Comment:
Hmm, I think something weird happened in the separation of the commits. Here you seem to remove the `MIN_BUCKET_FEERATE` constant, while in https://github.com/bitcoin/bitcoin/pull/13990/commits/c68f796c41b9b145c9c1b2c24aea505362ad1b63 you're adding it again.

Reply:
Prior to this PR, `MIN_BUCKET_FEERATE` defines both the lower bound of the buckets, and how they're aligned (that is, all buckets are at `FEE_SPACING**n * MIN_BUCKET_FEERATE`). This commit renames `MIN_BUCKET_FEERATE` to `BASE_BUCKET_FEERATE` because the alignment will continue to be based on it, even as the minimum gets reduced in the later commit that reintroduce `MIN_BUCKET_FEERATE` at a new, lower value.